From: Aaron Puchert <aaronpuchert@alice-dsl.net>
Date: Sat, 8 Apr 2023 23:41:04 +0200
Subject: [PATCH 04/20] Replace call to protected wxWindow::ProcessEvent by
 HandleWindowEvent

This is public in the base but protected in the derived class, a comment
indicates that HandleWindowEvent should be used instead.
---
 Source/tmwxGUI/tmwxCommon/tmwxApp.cpp | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/Source/tmwxGUI/tmwxCommon/tmwxApp.cpp b/Source/tmwxGUI/tmwxCommon/tmwxApp.cpp
index b9843fd..088d524 100644
--- a/Source/tmwxGUI/tmwxCommon/tmwxApp.cpp
+++ b/Source/tmwxGUI/tmwxCommon/tmwxApp.cpp
@@ -1133,7 +1133,7 @@ bool tmwxApp::ProcessEvent(wxEvent& event)
   wxWindow* topWindow = wxTheApp->GetTopWindow();
   if (mHelp && topWindow == (wxWindow*)(mHelp->mHtmlHelpFrame)) {
     eventStack.push_back(&event);
-    if (topWindow && topWindow->ProcessEvent(event)) {
+    if (topWindow && topWindow->HandleWindowEvent(event)) {
       eventStack.pop_back();
       return true;
     }
